api/spec/scanner-adapter-openapi-v1.1.yaml

```diff
@@ -68,6 +69,19 @@ paths:
         A non-blocking operation which enqueues a scan job and returns immediately. It returns a unique
         identifier which can be used to poll for generated scan reports by Harbor.
       operationId: AcceptScanRequest
+      parameters:
+        - name: X-Scan-Type
+          in: header
```

Comment:
I'm just curious why it is in the header rather than in the request body.

Reply:
The request body is also an option, the initial idea is to avoid changing the structure of the original scan request.

Reply:
I may be missing something, but what if adding an optional field to the scan request? It will not break anything as the scanners that don't support the new field just ignore it.

Reply:
Sure, we can add a field to the scan request if it can be more convenience than header from the scanner side.

Reply:
Actually, it doesn't matter to us. I was just wondering the reason. The Harbor team can decide it.

Reply:
Scan type is determined by the mine-type defined above. WHY do we still need to introduce this custom header for marking a new scan type?

Reply:
outdated.